Source/ACE.Common/NetworkSettings.cs

```diff
@@ -33,5 +34,14 @@ public class NetworkSettings
         [System.ComponentModel.DefaultValue(-1)]
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Populate)]
         public int MaximumAllowedSessionsPerIPAddress { get; set; }
+
+        /// <summary>
+        /// Will allow the given IP addresses to have unlimited sessions - recommend only use this for Admins
+        /// </summary>
+        /// <remarks>
+        /// </remarks>
+        [System.ComponentModel.DefaultValue(new string[] { })]
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Populate)]
+        public string[] AllowUlimitedSessionsFromIPAddresses { get; set; }
```

Comment:
rename to `AllowUnlimitedSessionsFromIPAddresses` ?
```suggestion
        [System.ComponentModel.DefaultValue(new string[] { })]
        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Populate)]
        public string[] AllowUnlimitedSessionsFromIPAddresses { get; set; }
```